src/javascript/app_2/Modules/SmartChart/Containers/smart_chart.jsx

```diff
@@ -48,6 +48,7 @@ class Chart extends React.Component {
                 symbol={this.props.symbol}
                 topWidgets={this.topWidgets}
                 isConnectionOpened={this.props.is_socket_opened}
+                showLastDigitStats={this.props.should_show_last_digit_stats}
```

Comment:
should also be `shouldShowLastDigitStats`, no?

Reply:
or that's not controlled by us? :(

Reply:
That is a smartchart prop